modules/javafx.graphics/src/main/java/javafx/scene/layout/GridPane.java

```diff
@@ -734,6 +734,20 @@ public GridPane() {
         getChildren().addListener((Observable o) -> requestLayout());
     }
 
+    /**
+     * Creates a GridPane layout with the given {@link #hgapProperty() hgap} and {@link #vgapProperty() vgap}.
+     *
+     * @param hgap the width of the horizontal gaps between columns
+     * @param vgap the height of the vertical gaps between rows
```

Comment:
I think that either
"the width of the gaps between columns" or
"the size of the horizontal gaps between columns"
otherwise it gives the illusion that there is a height for the horizontal gap.

Still reads fine as it is.

Reply:
I copied that mostly from the javadoc of `hgapProperty` and `vgapProperty`.
Tbh I like `the size of the horizontal gaps between columns` more as well. But then we are not consistent with the javadoc of the property. I'm fine either way, just that was my intention. :)

Reply:
I prefer the better phrasing over the consistent one :)